### ISSUE TITLE
Atom crashes on startup with jemalloc preloaded

### ISSUE THREAD

**Issue by ct16k** [atom/atom#8753]
Hello,

I'm unable to start Atom with jemalloc preloaded by means of /etc/ld.so.preload. The main window appears for a couple of seconds and then the app crashes. Console output is:
```
$ /usr/bin/atom: line 98:  4761 ***Segmentation fault      (core dumped)*** nohup "$ATOM_PATH" --executed-from="$(pwd)" --pid=$$ "$@" > "$ATOM_HOME/nohup.out" 2>&1
[4761:0911/125207:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
Gtk-Message: Failed to load module "pantheon-filechooser-module"
[4761:0911/125207:ERROR:browser_main_loop.cc(198)] GTK theme error: Unable to locate theme engine in module_path: "pixmap",
[4761:0911/125207:ERROR:browser_main_loop.cc(198)] GTK theme error: Unable to locate theme engine in module_path: "pixmap",
[4761:0911/125207:ERROR:browser_main_loop.cc(198)] GTK theme error: Unable to locate theme engine in module_path: "pixmap",
[4761:0911/125207:ERROR:browser_main_loop.cc(198)] GTK theme error: Unable to locate theme engine in module_path: "pixmap",
[4761:0911/125207:ERROR:browser_main_loop.cc(198)] GTK theme error: Unable to locate theme engine in module_path: "pixmap",
App load time: 201ms
ATTENTION: default value of option force_s3tc_enable overridden by environment.
[4795:0911/125208:INFO:renderer_main.cc(212)] Renderer process started
[4761:0911/125209:INFO:CONSOLE(11)] "activate linter-puppet", source: /home/ryan/.atom/packages/linter-puppet/lib/init.coffee (11)
Crash dump id: 1e66738c-49c4-4ce4-b267-ce8a2c10d891
Crash dump id: 9b64bbd7-b23c-41e6-b645-757e72e0f027
```
If I don't preload jemalloc when starting Atom, it works great.

I'm on elementary OS, using the .deb package on Atom's homepage.

**Comment by ct16k** [comment]
This issue seems to have gone away after updating to the newly released v1.1.0 (Atom now starts properly with libjemalloc.so preloaded, even after updating to jemalloc 4.0).

**Comment by 50Wliu** [comment]
:sparkles:
